package.json

```diff
@@ -4,14 +4,15 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf node_modules && yarn install --production=true && tsc && zip -r --exclude=*terraform* ./grader.zip index.js node_modules/ && yarn install --production=false",
+    "build": "rm -rf node_modules && rm -rf build && yarn install --production=true && tsc && cd build && zip -r --exclude=*terraform* ../dist/grader.zip index.js node_modules/ && cd .. && yarn install --production=false",
```

Comment:
```        zip warning: name not matched: node_modules/
zip I/O error: No such file or directory
zip error: Could not create output file (../dist/grader.zip)
```

This is big enough to be its own bash script. Could you make one and place it in `./scripts`? For reference, the [scripts directory over at frontend](https://github.com/source-academy/cadet-frontend/tree/master/scripts).